docs/release-notes/8.4.asciidoc

```diff
@@ -0,0 +1,74 @@
+[[release-notes-header-8.4.0]]
+== 8.4
+
+[discrete]
+[[release-notes-8.4.0]]
+=== 8.4.0
+
+[discrete]
+[[known-issue-8.4.0]]
+==== Known issue
+
+
+[discrete]
+[[breaking-changes-8.4.0]]
+==== Breaking changes
+// tag::breaking-changes[]
+// NOTE: The breaking-changes tagged regions are reused in the Elastic Installation and Upgrade Guide. The pull attribute is defined within this snippet so it properly resolves in the output.
+:pull: {pull}
+// end::breaking-changes[]
+
+[discrete]
+[[features-8.4.0]]
+==== Features
+* Creates a new rule type, New Terms, that fires when a value appears for the first time in a particular field ({pull}134526[#134526]).
+* Adds related alerts based on process ancestry to alert details ({pull}136009[#136009], {pull}138419[#138419], {pull}135340[#135340]).
+* Adds docs for Osquery API ({pull}137162[#137162]).
```

Comment:
```suggestion
```

Reply:
Removing because Osquery PRs are currently doc'd in the Kibana release notes. :)